### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by danraymond** [NicolasPetton/Projectify#16] [General]
@NicolasPetton ;

Could you please add "{{$:/plugins/nico/projectify/images/checkbox}} Projectify Check Box Toggle" or similar as a caption in the "$:/plugins/nico/projectify/ui/buttons/ToggleTodo" tiddler so its recognisable in the Control Panel's View Toolbar settings, otherwise its just a blank hyphen.

Thanks

**Comment by NicolasPetton** [comment]
Done, thansk!

I added a description as well.

**Comment by NicolasPetton** [comment]
I made a bug report issue here: https://github.com/NicolasPetton/Projectify/issues/17
